### DISCUSSION TITLE
Contribute to project

### DISCUSSION THREAD

**Discussion by itsevelynn21** [MultiMachineBuilder/MultiMachineBuilder#204] [General]
Hello! I would like to contribute to MultiMachineBuilder by working with some issues! If you're fine with that, let me know as soon as possible so I can inform you about which issues I'd like to fix!

**Comment by Monniasza** [comment]
@itsevelynn21 You can contribute to any issues that are not assigned, and you can also make pull requests, both as described in the CONTRIBUTING.MD file. I'd like you to report any issues to the applicable repo.

**Reply by itsevelynn21** [reply]
Could I work on issue #196 ? CONTRIBUTING.MD file says I need to have the issue assigned!

**Reply by Monniasza** [reply]
I'll need to discuss the new keys to use before you can implement them.
